frontend/src/feature/cards/composables/useCardsService.ts

```diff
@@ -26,30 +29,44 @@ const useCardsService = () => {
     }
   }
 
-  const getCategories = async (errorResult: string = 'throw') => {
-    const url = 'http://127.0.0.1:8080/api/categories';
+  const getCards = async (categoryId: string, page: number, errorResult: string = 'throw') : Promise<Card[]> => {
+    const url = apiUrl + 'cards?categoryId=' + categoryId + '&page=' + page;
```

Comment:
No string templates? I'm not against concatenation, just curious.

Reply:
yeah - funny I didn't use them here at the first place :-) Will change with next PR - it's nicer with template string